sources/netdog/src/cli/mod.rs

```diff
@@ -4,12 +4,12 @@ pub(crate) mod node_ip;
 pub(crate) mod set_hostname;
 pub(crate) mod write_resolv_conf;
 
-#[cfg(net_backend = "wicked")]
+#[cfg(feature = "wicked")]
 pub(crate) mod install;
-#[cfg(net_backend = "wicked")]
+#[cfg(feature = "wicked")]
 pub(crate) mod remove;
 
-#[cfg(net_backend = "systemd-networkd")]
+#[cfg(not(feature = "wicked"))]
```

Comment:
Thank you for making this change. I think this is better instead of having mutually-exclusive features like before.